### PR TITLE
fix: Fix rendering of RenderTargetBitmap

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): #9080

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by dr1rrb — COMMENTED** [review]
We need to add some a runtime test on this, we already had few encoding issues with the `RenderTragetBitmap`.

File: src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.Android.cs [hunk 1]
```diff
@@ -41,7 +41,21 @@ private static void SetSoftwareRendering(View nativeView, bool enabled)
 
 		private static ImageData Open(byte[] buffer, int bufferLength, int width, int height)
 		{
-			return ImageData.FromBitmap(BitmapFactory.DecodeByteArray(buffer, 0, bufferLength));
+			var bitmap = Bitmap.CreateBitmap(width, height, Bitmap.Config.Argb8888!)!;
```
[the remark this answers is not in the document]
Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.Android.cs:44]: That doesn't work. It's actually impossible to correctly create the bitmap without using the width and/or height. What we have in this code path isn't an encoded image, so I wouldn't expect any of the `Decode...` methods to work. :'(

[the remark this answers is not in the document]
Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.Android.cs:44]: There is `Bitmap.CreateBitmap` overload that could work, but it takes `int[]` not `byte[]`. Maybe I could try some funny `unsafe` code. Would that be interesting to do?

**Comment by Youssef1313** [comment]
> We need to add some a runtime test on this, we already had few encoding issues with the `RenderTragetBitmap`.

The change is already captured by the unskipped test.

File: src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.Android.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.Android.cs:44]: @dr1rrb Can you check the new code? I'm not feeling so good about it, but it's probably safe if passed CI.

**Review by dr1rrb — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.Android.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (dr1rrb) [reply src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.Android.cs]: CI seems to go green, so LGTM

**Pull request merged**